src/pieces/commandManager.ts

```diff
@@ -166,38 +197,38 @@ export async function loadCommands(bot: Client): Promise<void> {
 
 	await Promise.all(awaitedCmds);
 
-	let permsUpdated = 0;
-	console.log('Checking for updated permissions...');
-	await Promise.all(commands.cache.map(async command => {
-		let curPerms: ApplicationCommandPermissionData[];
-		try {
-			curPerms = await command.permissions.fetch({ command: command.id });
-		} catch (err) {
-			curPerms = [];
-		}
+	// TEMPORARILY DISABLED DUE TO DJS UPDATE
+	// bots cannot set their own perms as of djs v13.6
 
-		const botCmd = bot.commands.find(cmd => cmd.name === command.name);
-		if (botCmd
-			&& (botCmd.permissions.length !== curPerms.length
-				|| !botCmd.permissions.every(perm =>
-					curPerms.find(curPerm => isPermissionEqual(curPerm, perm))))) {
-			console.log(`Updating permissions for ${botCmd.name}`);
-			permsUpdated++;
-			return commands.permissions.set({
-				command: command.id,
-				permissions: botCmd.permissions
-			});
-		}
-	}));
+	// let permsUpdated = 0;
```

Comment:
If we end up merging your duc tape, we should get rid of this comment block

Reply:
this is the former permissions manager, i wasn't too keen on removing this in case there is a loophole around the new perms system or related, but i suppose git exists for a reason

Reply:
dead code should just always be removed before PRing, for the exact reason that you just stated: we can always get the code back by looking at the older commits on GH